jupyter_resource_usage/api.py

```diff
@@ -113,17 +105,18 @@ async def get(self, matched_part=None, *args, **kwargs):
         usage_request = session.msg("usage_request", {})
 
         control_channel.send(usage_request)
-        poller = zmq.Poller()
+        poller = zmq.asyncio.Poller()
         control_socket = control_channel.socket
         poller.register(control_socket, zmq.POLLIN)
-        for i in range(1, MAX_RETRIES + 1):
```

Comment:
I don't think a second try could ever be taken, because the options are:

1. `events` is empty, in which case empty dict is written, or
2. `events` contains the control socket, in which case the response is written

both of which end with `break`

There shouldn't be a way for the `continue` branch to ever be taken, which meant that the loop could only ever take one iteration.